managedtenants/core/addons_loader/sss.py

```diff
@@ -134,6 +135,8 @@ def _walk(self, data):
                     res["sss_delete"] = self._walk_sss(item)
                 else:
                     res["sss_deploy"] = self._walk_sss(item)
+            elif item["kind"] == TEMPLATE_KIND:
+                res["template_deploy"]= self._walk_template(item)
```

Comment:
```suggestion
                template_sss = item["objects"][0]
                walked_template_sss = self._walk_sss(child_sss)
                res["template_deploy"][0] = walked_template_sss
```
I didn't try interpreting this code but I guess we can re-use the `_walk_sss` method here :)